resources/user-agents/crawlers/site-monitors/catchpoint.json

```diff
@@ -0,0 +1,39 @@
+{
+  "division": "Catchpoint Monitoring Service",
+  "sortIndex": 490,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Catchpoint Monitoring Service",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Catchpoint Monitoring Service",
+        "Browser": "Catchpoint Monitoring Service",
+        "Browser_Maker": "Catchpoint Systems, Inc.",
+        "Browser_Type": "Bot/Crawler",
+        "Frames": "true",
+        "IFrames": "true",
+        "Tables": "true",
+        "Cookies": "true",
+        "JavaScript": "true",
+        "JavaApplets": "false",
+        "CssVersion": "3",
+        "Crawler": "true"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#Catchpoint)*applewebkit* (*khtml*like*gecko*) Chrome/*Safari/*",
+          "engine": "Blink",
+          "device": "Windows Desktop",
+          "platforms": [
+            "Win7_x64", "Win7_64", "Win7_32",
```

Comment:
I tried looking in our logs to see if these various windows versions were necessary or not, but we haven't seen this particular bot.